website/documentation/050-tutorials/content/howto/maintain-shoot/_index.md

```diff
@@ -11,94 +11,154 @@ aliases: ["readmore/shoot-maintain"]
+You can either specify the maintenance window in the shoot spec (`.spec.maintenance.timeWindow`) or the start time of the maintenance window using the Gardener dashboard (**CLUSTERS** > **[YOUR-CLUSTER]** > **OVERVIEW** > **Lifecycle** > **Maintenance**).
+
+### Auto-Update and Forceful Updates
+
+To trigger updates during the maintenance window automatically, Gardener offers the following methods:
+
+* _Auto-update_: <br>Gardener starts an update during the next maintenance window whenever there’s a version available in the `CloudProfile` that is higher than the one of your shoot cluster specification, and that isn’t classified as `preview` version. For Kubernetes versions, auto-update only updates to higher patch levels.
+  
+  You can either activate auto-update on the Gardener dashboard (**CLUSTERS** > **[YOUR-CLUSTER]** > **OVERVIEW** > **Lifecycle** > **Maintenance**) or in the shoot cluster specification:
+  
+  *  `.spec.maintenance.autoUpdate.kubernetesVersion: true`
+  *  `.spec.maintenance.autoUpdate.machineImageVersion: true`
+
+* _Forceful updates_: <br>In the maintenance window, Gardener compares the current version given in the shoot cluster specification with the version list in the `CloudProfile`. If the version has an expiration date and if the date is before the start of the maintenance window, Gardener starts an update to the highest version available in the `CloudProfile` that isn’t classified as `preview` version. The highest version in `CloudProfile` can’t have an expiration date. For Kubernetes versions, Gardener only updates to higher patch levels or consecutive minor versions. 
+
+If you don’t want to wait for the next maintenance window, you can annotate the `Shoot` specification with `shoot.gardener.cloud/operation: maintain`. Gardener then checks immediately if there’s an auto-update or a forceful update needed.
+
+{{% notice info %}}
+<p>
+Forceful version updates are even executed if the auto-update for the Kubernetes version, or the auto-update for the machine image version is deactivated (set to `false`).
+</p>
 {{% /notice %}}
 
+With expiration dates, administrators can give shoot cluster owners more time for testing before the actual version update happens, which allows smoother transitions to new versions.
 
-### Forceful Version Updates
+### Kubernetes Update Paths
 
-While the [automatic Kubernetes](#kubernetes-automatic-version-updates) and [operating system updates](#automatic-operating-system-version-updates), are an opt-in feature for the Shoot cluster owner, Gardener administrators can force patch and minor version updates.
+The bigger the delta of the Kubernetes source version and the Kubernetes target version, the better it must be planned and executed by operators. Gardener only provides automatic support for updates that are uncritical for your deployed resources:
```

Comment:
critical != harmful, patch update can be critical (for example includes CVE fixes).

```suggestion
The bigger the delta of the Kubernetes source version and the Kubernetes target version, the better it must be planned and executed by operators. Gardener only provides automatic support for updates that are not harmful for the cluster workload:
```

Reply:
@ialidzhikov: Thanks for your review (working on it)!

"harmful" does sound weird (I checked with a native speaker), but I see your point.

Proposal:

Gardener only provides automatic support for updates that can be applied safely to the cluster workload.

Other idea: ...that can be applied without disrupting the cluster workload...